### PR TITLE
Sci bumps

### PR DESCRIPTION
**Pull request by jsmolic**

### CODE REVIEW

File: sci-libs/nfft/nfft-3.5.2.ebuild [hunk 1]
```diff
@@ -34,16 +34,19 @@ pkg_pretend() {
...
+		--disable-static \
+		$(use_enable openmp)
 }
 
 src_install() {
 	default
-	use doc || rm -r "${ED}/usr/share/doc/${P}/html" || die
 
-	# infft uses pkg-config to record its private dependencies
+	if use !doc; then
```
Comment (SoapGentoo) [inline sci-libs/nfft/nfft-3.5.2.ebuild:46]: prefer bash negation over use negation, i.e., to `if ! use doc; then`

File: sci-libs/nfft/nfft-3.5.2.ebuild [hunk 2]
```diff
@@ -34,16 +34,19 @@ pkg_pretend() {
...
+		rm -r "${ED}/usr/share/doc/${P}/html" || die
```
Comment (SoapGentoo) [inline sci-libs/nfft/nfft-3.5.2.ebuild:47]: move the quote?

File: sci-misc/h5utils/h5utils-1.13.1.ebuild [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~x86 ~amd64-linux ~x86-linux"
+IUSE="hdf octave"
+
+DEPEND="
+	media-libs/libpng:0=
+	sci-libs/hdf5:0=
+	sys-libs/zlib:0=
```
Comment (SoapGentoo) [inline sci-misc/h5utils/h5utils-1.13.1.ebuild:20]: remove the subslot operator on zlib

**Review by SoapGentoo — CHANGES_REQUESTED** [review]

File: sci-misc/h5utils/h5utils-1.13.1.ebuild (hunk 3 above)
Comment (jsmolic) [reply sci-misc/h5utils/h5utils-1.13.1.ebuild]: Good catch, done

File: sci-libs/nfft/nfft-3.5.2.ebuild (hunk 1 above)
Comment (jsmolic) [reply sci-libs/nfft/nfft-3.5.2.ebuild]: Done

File: sci-libs/nfft/nfft-3.5.2.ebuild (hunk 2 above)
Comment (jsmolic) [reply sci-libs/nfft/nfft-3.5.2.ebuild]: Yes

**Review by SoapGentoo — APPROVED** [review]

**Pull request closed without merging**